bs-react-native-next/src/Style.re

```diff
@@ -252,6 +252,12 @@ external style:
   t =
   "";
 
-// Duplicated from StyleSheet.re for convenience.
-[@bs.module "react-native"] [@bs.scope "StyleSheet"]
+[@deprecated "Use Style.array([|t|]) instead"]
+[@bs.module "react-native"]
+[@bs.scope "StyleSheet"]
 external flatten: array(t) => t = "";
+
+external array: array(t) => t = "%identity";
+external arrayOption: array(option(t)) => t = "%identity";
+external list: list(t) => t = "%identity";
+external listOption: list(option(t)) => t = "%identity";
```

Comment:
It's great that this works out of the box. I wonder whether there is any performance difference to using StyleSheet.flatten.

Reply:
StyleSheet.flatten (per doc) seems to have a cost since it's resolve stylesheets references (int) to the styles associated etc (see 
https://facebook.github.io/react-native/docs/stylesheet#flatten).

So passing styles from array should be better as ids are directly passed to the bridge, with eventual dynamic styles.